edb/schema/pointers.py

```diff
@@ -865,7 +865,7 @@ def _parse_computable(
         expr: qlast.Base,
         schema: s_schema.Schema,
         context: sd.CommandContext,
-    ) -> Tuple[s_schema.Schema, s_types.Type, Optional[PointerLike]]:
+    ) -> Tuple[s_schema.Schema, s_types.TypeShell, Optional[PointerLike]]:
```

Comment:
(What's a type shell?)

Reply:
When we build a schema delta IR tree, we cannot refer to actual schema objects directly, because they might not exist until we actually run the IR.  So, we use `ObjectShell` and its derivatives.  It's somewhat analogous to `TypeRef` in EdgeQL IR, except we don't usually need to fully describe the underlying object in a `Shell`, only the means to properly look it up in the schema once available.